### DISCUSSION TITLE
ZHEMI101 Powermeter change LED pulse

### DISCUSSION THREAD

**Discussion by pimboli** [Koenkk/zigbee2mqtt#8405] [Devices]
Hi, 

I successfully added the ZHEMI101 powermeter to Zigbee2QTT / Home Assistant.
But I can't find where I can configure the LED pulse rate.

I found on the internet a PDF with a command to define the pulse rate.
Seems that by default its set to 1000 and I need to set it to 400.

<img width="580" alt="Notification_Center" src="https://user-images.githubusercontent.com/5020679/129698814-26f882e2-57e4-4a14-a373-fcfea3b1fa4e.png">

Thank you in advance

**Comment by pimboli** [comment]
I saw on deconz they added some support:
https://github.com/dresden-elektronik/deconz-rest-plugin/issues/4337

Would be great if someone could help me to add the PulseConfiguration setting also here on zigbee2mqtt :)

**Comment by grudyjsy** [comment]
Hi,
I have the same issue but in my case I need to set the pulse to 3200. Is there a way to modify the calculation of a sensor or create a new sensor looking at this one ? i.e. I could get the sensor value / 3.125 to get back to the approximate correct value.
I'm new to home assistant so still learning and going through forums etc.
It would be great to get this fixed, now that home assistant are putting a lot of their efforts into the energy monitoring section of the system.
Thanks

**Comment by robertalexa** [comment]
Hey guys,

Let close this conversation and move it into the Issue https://github.com/Koenkk/zigbee2mqtt/issues/8638

I do not have the device but I will do my best to help you guys out.
